packages/cli/generators/model/templates/model.ts.ejs

```diff
@@ -1,7 +1,12 @@
-import {Entity, model, property} from '@loopback/repository';
+<% if (isModelBaseBuiltin) { -%>
+import {<%= modelBaseClass %>, model, property} from '@loopback/repository';
+<% } else { -%>
+import {model, property} from '@loopback/repository';
+import {<%= modelBaseClass %>} from '.';
```

Comment:
nitpick: Would it be better to import from the base class file directly?

Reply:
Isn't it the reason for the index.ts file in src/models?